templates/components/status.html.twig

```diff
@@ -1,16 +1,27 @@
-{% macro status(text, options) %}
-    {% set _color = options.color ?? 'green' %}
-    {% set _animated = options.animated ?? true %}
-    {% set _lite = options.lite is defined ? options.lite  : false %}
-    {% set _with_dot = options.with_dot is defined ? options.with_dot  : false %}
-    {% set _extraClass = options.extraClass ?? '' %}
-
-    {%- apply spaceless -%}
-        <span class="status status-{{ _color }} {{ _lite ? 'status-lite' : '' }} {{ _extraClass }}">
-        {% if _with_dot %}
-            <span class="status-dot {{ _animated ? 'status-dot-animated' : '' }}"></span>
-        {% endif %}
-            {{ text }}
-        </span>
-    {%- endapply -%}
-{% endmacro %}
+{% macro status(text, options) %}
+    {% import '@Tabler/includes/utils.html.twig' as utils %}
+    {% import '@Tabler/components/status_dot.html.twig' as status_dot %}

```

Comment:
Is this ok to import only inside a `if`?

Like that:
```twig
{% if _withDot %}
   {% import '@Tabler/components/status_dot.html.twig' as status_dot %}
   {{ status_dot({color: _color, animated : _animated}) }}
{% endif %}
```

Or it is too much?

Reply:
Not worth it IMO. 
I am not entirely sure, but I believe it will be imported anyway during compilation.